includes/Media/SVG.php

```diff
@@ -274,7 +274,7 @@ public function wp_handle_upload( array $upload ): array {
 	 *
 	 * @phpstan-param array{ext?: string, type?: string, proper_filename?: bool} $wp_check_filetype_and_ext
 	 */
-	public function wp_check_filetype_and_ext( $wp_check_filetype_and_ext, $file, $filename, $mimes, $real_mime ): array {
+	public function wp_check_filetype_and_ext( array $wp_check_filetype_and_ext, string $file, string $filename, array $mimes, $real_mime ): array {
```

Comment:
$real_mine type?

Reply:
`$real_mime` can be a bool or a string, so can't use a type hint here. And union type hints are not supported on PHP 7.4